pkg/ebpf/ebpftest/bpfdebug_unsupported.go

```diff
@@ -10,17 +10,17 @@ package ebpftest
 import "testing"
 
 // LogTracePipe is unsupported
-func LogTracePipe(t *testing.T) {
+func LogTracePipe(*testing.T) {
 }
 
 // LogTracePipeSelf is unsupported
-func LogTracePipeSelf(t *testing.T) {
+func LogTracePipeSelf(*testing.T) {
 }
 
 // LogTracePipeProcess is unsupported
-func LogTracePipeProcess(t *testing.T, pid uint32) {
+func LogTracePipeProcess(_ *testing.T, pid uint32) { // nolint:revive // TODO fix revive unused-parameter
```

Comment:
For this function, I'm OK with removing the argument names since this is the unsupported stub file.